data/remnant/remnant 2 side missions.txt

```diff
@@ -715,3 +715,47 @@ mission "Remnant: Scent of Rain 1"
 			`	He shakes his head. "But no matter. Planning for the storm is my duty, not yours, and the front is yet to appear.`
 			label scanningpayment
 			`	"We have allocated you <payment> for the risks undertaken. Please use it wisely. While it is my duty to anticipate threats to us all, it is your duty to ensure you are prepared to face the winds they may bring." He turns and strides away, quickly vanishing into the tunnels beneath the spaceport.`
+
+mission "Remnant: Wont Someone Please Think Of The Children"
+	landing
+	name "Sympathy For The Korath"
+	description "Travel to the <waypoints> system to fight and disable a stranded Ra'gru Ik 618. Do not destroy or capture the ship. Our intel says the world ship has a large civilian population."
+	source
+		government "Remnant"
+		not attributes "station"
+	waypoint "Parca"
+	to offer
+		has "event: remnant: cognizance calm"
+		"combat rating" > 8103
+	on offer
+		conversation
+			`You step out of your airlock to find Prefect Chilia waiting for you with a worried look on his face. "Captain <first> I have a difficult request of you."`
+			`	"Our scouts are reporting a stranded Korath worldship in Parca, a Ra'gru Ik 618 to be precise. Normally we would request you track down this ship and eliminate, but our scouts also reported that this appears to be home to a large civilian population."`
+			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arranged for a prisoner swap with the Korath."`
```

Comment:
Now, whilst I like the idea of it, I feel like remnant - Korath interactions are based on things that are not said, not explicit arrangements, although they both don't want to harm the other too much, and will try to find the best ways to approach things. So, I'd make it more like this:
```suggestion
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we will find a way to bring them home."`
```
But don't change this until Zitchas gets to look at it

Reply:
I think this is more in keeping with the way Zitchas envisions these encounters?

```suggestion
			`	"Would you be up for disabling this Korath worldship, so the threat is eliminated while preserving the lives of the innocents on board? After you have done so we can arrange to have the ship disarmed and sent back to a Korath station."`
```

Reply:
Thats the thing, how do you bring it back? You fix the JD? Give it another one? That's why I kept it vague

Reply:
Oh, right, I hadn't thought of *that* aspect.  This will require some thought when I've not stayed up through the night again.

Reply:
Added a requirement that you have completed a total of at least 10 Broken Jump drive missions, and changed this that the Remnant will disarm the ship and then fix their jump drive, so the Korath will hopefully decide to just go home.